lib/plugins/aws/utils/request.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
```

Comment:
Let's put it into `lib/aws/request.js`

Reply:
I'm not sure about the path here, is it `serverless/lib/aws` or `serverless/lib/plugins/aws/lib`? Because the first option would mean a new directory and it would also break the plugin isolation a bit.

Reply:
> Because the first option would mean a new directory and it would also break the plugin isolation a bit.

It's `lib/aws/request`. Note it's not a plugin anymore, also everything in `lib` should be considered private

Reply:
Ok :smile: will do :smile: thanks. Just to try and understand the big plan here, is there a plan to move aws outside of the plugin dir?

Reply:
Utils will rather be moved here, but it's an ongoing progress. It might be even that some point Framework will get modularized and AWS stuff will be moved to other repository (but that's a long term consideration, nothing we will rush with now)

Reply:
Thanks for the answer :smile: I've moved it there in the last commit.